clientServiceClient/api.go

```diff
@@ -57,6 +57,11 @@ type ClientRegisterInfo struct {
 	SigningKeys map[string]string `json:"signing_key,omitemtpy"`
 }
 
+// AdminClientSigningKeyPatchRequest wraps the information needed to patch signing keys
+type AdminClientSigningKeyPatchRequest struct {
+	SigningKeys map[string]string `json:"signing_key"`
```

Comment:
reminder for an api spec, can we tag this `signing_keys` and add naming comments to indicate this is the PublicSigning Key